src/main/java/io/rpg/controller/Controller.java

```diff
@@ -176,6 +176,10 @@ public void onMouseClickedEvent(MouseClickedEvent event) {
         popupController.openTextImagePopup("Picked up an item!", objectView.getImage(), getWindowCenterX(), getWindowCenterY());
         objectView.setVisible(false);
       }
+
+      if (object instanceof DialogGameObject) {
```

Comment:
for now fine, but there won't be DialogGameObject etc in the future, instead we'll have one GameObject with different onAction()
I'd add a comment like // TODO: change to generic GameObject with proper onAction()